.stylelintrc

```diff
@@ -12,7 +12,7 @@
         "color-named": "never",
         "comment-empty-line-before": null,
         "declaration-no-important": true,
-        "declaration-block-trailing-semicolon": "never",
+        "declaration-block-trailing-semicolon": "always",
```

Comment:
was throwing an error I couldn't get rid of and it's probably a good idea to always have semicolons for css